tests/speed_tests.py

```diff
@@ -52,22 +54,49 @@ def paintEvent(self, event):
 
 
 def main():
-    frac = fractal.Fractal()
+    print("Testing SIMPLE BAND LAYER")
+
+    print("C calculations: ")
+    test_c_by_size(fractal.SimpleBandsLayer(), 800, 650, 100)
+    print("-----------------------------")
+    test_c_by_size(fractal.SimpleBandsLayer(), 1000, 1000, 50)
+    print("-----------------------------")
+    test_c_by_size(fractal.SimpleBandsLayer(), 5000, 5000, 15)
+    print("-----------------------------")
+    test_c_by_size(fractal.SimpleBandsLayer(), 10000, 10000, 5)
+
+    print()
+
+    print("Testing SMOOTH BAND LAYER")
```

Comment:
I think testing different layers should be different unit tests to improve granularity